README.rst

```diff
@@ -34,3 +34,20 @@ Interactive utility for writing CPython ``Misc/NEWS.d`` entries. See
 the blurb_ directory for more details.
 
 .. _blurb: https://github.com/python/core-workflow/tree/master/blurb
+
+
+Other core workflow tools
+-------------------------
+
++------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------+-------------------------------------------------------+
```

Comment:
I think we can use the simpler syntax at http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables

Reply:
Also, I might be wrong, but only the first column of a row should match the length of its header. For example, the following table is valid in reST:

```rst
============== =============
Name           Issue tracker
============== =============
Some long name This is a very long URL of the issue tracker
Some long name https://www.python.org/
============== =============
```

That would simplify markup of the table. See the latest example at http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables for more details.